frontend/src/components/ui/alert.ts

```diff
@@ -15,6 +15,7 @@ export class Alert extends LitElement {
   @property({ type: String })
   variant: "success" | "warning" | "danger" | "info" = "info";
 
+  // postcss-lit-disable-next-line
```

Comment:
Why do we need to disable postcss-lit? I'm not seeing a visible difference disabling/enabling.

Reply:
I don't know if we do — I'd put this disable everywhere I wasn't directly working on because I didn't want it to break anything unintentionally